.travis.yml

```diff
@@ -15,6 +15,7 @@ before_script:
   # Create conda virtual environments
   - conda env create -f envs/main.yml
   - conda env create -f envs/dev.yml
+  - conda env create -f envs/plot_heatmap_of_MotEvo_results.yml
```

Comment:
Yes, that was exactly what I had in mind. Could you please add building all the other conda v-envs as well? All of those which *snakemake* builds automatically.

Reply:
OK, now I notice that the path is wrong here as well. You have to put all the YAML recipes for snakemake rules v-envs under `envs` of the `workflow`, not under the root-level directory `envs`.